cvat/apps/auto_annotation/views.py

```diff
@@ -0,0 +1,314 @@
+        )
+
+        if result is None:
+            slogger.glob.info('auto annotation for task {} canceled by user'.format(tid))
+            return
+
+        annotation.clear_task(tid)
+        annotation.save_task(tid, result)
+        slogger.glob.info('auto annotation for task {} done'.format(tid))
+    except:
+        try:
+            slogger.task[tid].exception('exception was occured during auto annotation of the task', exc_info=True)
+        except:
+            slogger.glob.exception('exception was occured during auto annotation of the task {}'.format(tid), exc_into=True)
+
+@login_required
+def get_meta_info(request):
+    try:
+        queue = django_rq.get_queue('low')
+        tids = json.loads(request.body.decode('utf-8'))
+        result = {}
+        for tid in tids:
+            job = queue.fetch_job('auto_annotation.create/{}'.format(tid))
+            if job is not None:
+                result[tid] = {
+                    "active": job.is_queued or job.is_started,
+                    "success": not job.is_failed
+                }
+
+        return JsonResponse(result)
+    except Exception as ex:
+        slogger.glob.exception('exception was occurred during auto annotation meta request', exc_into=True)
+        return HttpResponseBadRequest(str(ex))
+
+@login_required
+@permission_required(perm=['engine.task.change'],
+    fn=objectgetter(TaskModel, 'tid'), raise_exception=True)
+def create(request, tid):
+    slogger.glob.info('auto annotation create request for task {}'.format(tid))
+
+    def write_file(path, file_obj):
+        with open(path, 'wb') as upload_file:
+            for chunk in file_obj.chunks():
+                upload_file.write(chunk)
+
+    try:
+        db_task = TaskModel.objects.get(pk=tid)
+        upload_dir = db_task.get_upload_dirname()
+        queue = django_rq.get_queue('low')
+        job = queue.fetch_job('auto_annotation.create/{}'.format(tid))
+        if job is not None and (job.is_started or job.is_queued):
+            raise Exception("The process is already running")
+
+        params = request.POST.dict()
+        model_file = request.FILES['model']
+        model_file_path = os.path.join(upload_dir, model_file.name)
+        write_file(model_file_path, model_file)
+
+        weights_file = request.FILES['weights']
+        weights_file_path = os.path.join(upload_dir, weights_file.name)
+        write_file(weights_file_path, weights_file)
+
+        config_file = request.FILES['config']
+        config_file_path = os.path.join(upload_dir, config_file.name)
+        write_file(config_file_path, config_file)
+
+        convertation_file = request.FILES['conv_script']
+        convertation_file_path = os.path.join(upload_dir, convertation_file.name)
+        write_file(convertation_file_path, convertation_file)
+
+
+        db_labels = db_task.label_set.prefetch_related('attributespec_set').all()
+        db_labels = {db_label.id:db_label.name for db_label in db_labels}
+
```

Comment:
I think we need to confirm an user immediately:
I know, this check will later, but if queue is busy, user can waiting long for bad response. 
```
if not db_labels:
    return HttpResponseBadRequest('No labels found for an auto annotation')
```

Reply:
UPD: in this case we need move an labels intersection search here